src/api/index.ts

```diff
@@ -1,9 +1,41 @@
-import axios from "axios";
-let headers = {};
+const BASE_URL = "https://api.campus.kpi.ua";
 
-export const Http = axios.create({
-  baseURL: "https://api.campus.kpi.ua/",
-  headers,
-});
+const defaultHeaders = {
+  "Content-Type": "application/json",
+};
+
+interface FetchOptions extends RequestInit {
+  headers?: Record<string, string>;
+}
+
+const fetchWrapper = async (post_url: string, options: FetchOptions = {}) => {
```

Comment:
- Follow naming conventions: `post_url` -> `postUrl`
- Please choose better naming for this parameter (something like `requestUrl`)

Reply:
Thank you.
Renamed to requestUrl